### PR TITLE
add cp/ edge to supported libs.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -179,6 +179,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	edge: {
+		prettyLabel: "Edge",
+		repoName: "Edge",
+		repoUrl: "https://github.com/cartesia-ai/edge",
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:185]: Is this a "soon to be released" repository?

File: packages/tasks/src/model-libraries-snippets.ts [hunk 2]
```diff
@@ -170,6 +170,21 @@ export const diffusers = (model: ModelData): string[] => {
 	}
 };
 
+export const edge = (model: ModelData): string[] => [
+	`# pip install --no-binary :all: cartesia-pytorch
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:174]: Yes

File: packages/tasks/src/model-libraries-snippets.ts [hunk 3]
```diff
@@ -170,6 +170,21 @@ export const diffusers = (model: ModelData): string[] => {
...
+from cartesia_pytorch import ReneLMHeadModel
+from transformers import AutoTokenizer
+
+model = ReneLMHeadModel.from_pretrained("${model.id}").half().cuda()
+tokenizer = AutoTokenizer.from_pretrained("allenai/OLMo-1B-hf")
+in_message = ["Rene Descartes was"]
+inputs = tokenizer(in_message, return_tensors="pt")
+outputs = model.generate(inputs.input_ids.cuda(), max_length=50, top_k=100, top_p=0.99)
+out_message = tokenizer.batch_decode(outputs, skip_special_tokens=True)[0]
+print(out_message)
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:184]: I think it is good to leave it as-is for now. Will adapt as they release more.

File: packages/tasks/src/model-libraries.ts [hunk 4]
```diff
@@ -179,6 +179,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	edge: {
```
[the remark this answers is not in the document]
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:182]: No clue for now, waiting for more clarity.

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:185]: Yes.

File: packages/tasks/src/model-libraries.ts [hunk 5]
```diff
@@ -128,7 +128,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.bm25s,
 		filter: false,
 		countDownloads: `path:"params.index.json"`,
-	},
+	},
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:131]:
```suggestion
	},
```

File: packages/tasks/src/model-libraries.ts [hunk 6]
```diff
@@ -179,6 +179,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	edge: {
+		prettyLabel: "Edge",
+		repoName: "Edge",
+		repoUrl: "https://github.com/cartesia-ai/edge",
+		snippets: snippets.edge,		
+	},
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:187]:
```suggestion
	},
```

**Review by pcuenca — APPROVED** [review]
lgtm with the comments discussed

File: packages/tasks/src/model-libraries.ts [hunk 7]
```diff
@@ -179,6 +179,18 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	cartesia_pytorch: {
+		prettyLabel: "Cartesia Pytorch",
+		repoName: "Cartesia Pytorch",
+		repoUrl: "https://github.com/cartesia-ai/cartesia_pytorch",
+		snippets: snippets.cartesia_pytorch,
+	},
+	cartesia_mlx: {
```
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries.ts:188]: I'm unsure if this is the correct `library_name` - I will wait for confirmation before the merge.

File: packages/tasks/src/model-libraries-snippets.ts [hunk 8]
```diff
@@ -170,6 +170,45 @@ export const diffusers = (model: ModelData): string[] => {
...
+from cartesia_pytorch import ReneLMHeadModel
+from transformers import AutoTokenizer
+
+model = ReneLMHeadModel.from_pretrained("${model.id}")
+tokenizer = AutoTokenizer.from_pretrained("allenai/OLMo-1B-hf")
+
+in_message = ["Rene Descartes was"]
+inputs = tokenizer(in_message, return_tensors="pt")
+
+outputs = model.generate(inputs.input_ids.cuda(), max_length=50, top_k=100, top_p=0.99)
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries-snippets.ts:184]:
```suggestion
outputs = model.generate(inputs.input_ids, max_length=50, top_k=100, top_p=0.99)
```

File: packages/tasks/src/model-libraries-snippets.ts [hunk 9]
```diff
@@ -170,6 +170,45 @@ export const diffusers = (model: ModelData): string[] => {
...
+print(out_message)
+)`,
+];
+
+export const cartesia_mlx = (model: ModelData): string[] => [
+	`import mlx.core as mx
+import cartesia_mlx as cmx
+
+model = cmx.from_pretrained("${model.id}")
+model.set_dtype(mx.float32)
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries-snippets.ts:196]: is this necessary? (just curious)

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:196]: According to their official snippet, yes! but I haven't been able to test it till now.

**Comment by Vaibhavs10** [comment]
Merging for now, will make follow-up updates as it comes!

**Pull request merged**